README.md

```diff
@@ -177,3 +177,13 @@ In case you have an existing Torcx image you can convert it with the `convert_to
 ```
 
 Please make also sure that your don't have a `containerd.service` drop in file under `/etc` that uses Torcx paths.
+
+## For maintainers: how to release?
+
+CI can be kicked-off by overriding the `latest` tag. The `latest` release artifacts will be updated consequently here: https://github.com/flatcar/sysext-bakery/releases/tag/latest
+```
+git rebase origin/main
+git tag -d latest
+git tag -as latest
+git push origin --force latest
+```
```

Comment:
Or maybe we can change the CI to re-release on every `main` change

Reply:
Would make sense to me. I'm working on a build automation feature to build releases based on a list of versions in a text file, and include all sysexts from the previous "latest" release. Your proposal would be a great follow-up task to that PR.

Reply:
==> https://github.com/flatcar/sysext-bakery/pull/40

Reply:
Also #42 .